core/src/test/java/io/confluent/rest/SslTest.java

```diff
@@ -226,6 +227,36 @@ public void testHttpsOnly() throws Exception {
     }
   }
 
+  @Test
```

Comment:
[minor] can this test be parameterized on tls or ssl versions?

Reply:
REST logic to restrict cipher suites is agnostic of protocol, so parameterising on ssl version is unnecessary at the moment.

Reply:
> is agnostic of protocol
that's why paramterize tests on ssl version proves this statement, what do you think?

Reply:
Parameterised test would actually add more tests. That signals its important to test for different tls version 1/2/3. 

Using "tlsv1.3" is accidental & orthogonal to the behaviour being tested. Only needed for setup, better to add that as a comment, done now.

Reply:
> Parameterised test would actually add more tests. 

That would be fine, tests in this class running fast, so we would not mind having more tests.

> That signals its important to test for different tls version 1.1/2/3.
This is actually the point of parameterize tests, because there is some cipher that is only applicable for tls version 1.3 as shown in this test. That would help us when we actually restrict the cipher because we would know the error if the restricted cipher isn't compatible with the protocol.